tvb_library/tvb/basic/logger/builder.py

```diff
@@ -95,3 +99,7 @@ def get_logger(parent_module=''):
     :param parent_module: module name for which to create logger.
     """
     return GLOBAL_LOGGER_BUILDER.build_logger(parent_module)
+
+
+def set_loggers_level(level):
```

Comment:
Could you pls adjust 1-2 demo notebook with this function usage (e.g. `compare_connectivity_normalization`,  `using_your_own_connectivity`) plus maybe even a dedicated text cell in each to explain its purpose ?

Also consider writing some docstring on this function.